README.md

```diff
@@ -661,6 +661,31 @@ The following demonstrates the `suffix` directive which specifies that `X-Forwar
   - [ X-Forwarded-For, { value: 2, as: suffix } ]
 ```
 
+Proxy Verifier supports two rule modifications to the six directives:
+
+Directive  | Description
+---------  | ------------
+not        |  Inverts the result of a rule.
+nocase     |  Modifies a rule to ignore any distinctions between upper and lower case letters.
+
+The following demonstrates the `prefix` directive which specifies that `X-Forwarded-For` should have been received from the proxy with a field value not starting with "a":
```

Comment:
Good docs.

Reply:
Done. Not sure if there needed to be any changes to the non-yaml part of the test.